test/transform.js

```diff
@@ -23,7 +23,17 @@ test('transform adds sourcemap comment and uses cache on second time', function
 
     var es6ify = proxyquire('..', { './compile' : trackingCompile } )
 
-    var file = path.join(__dirname, '../example/src/features/iterators.js');
+    var sourceRoot = path.join(__dirname, '..', 'example', 'src');
+    var relPath = path.join('features', 'iterators.js');
+
+    var opts = {sourceRoot: sourceRoot};
+
+    es6ify = es6ify.configure(opts);
+
+    var file = path.join(sourceRoot, relPath);
+    if (! opts.sourceRoot) relPath = path.basename(file);
```

Comment:
Oh, character-counting is how you measure verbosity, OK. And yes, falsy values should not trigger defaults.